docs/triaging.md

```diff
@@ -43,6 +43,15 @@ categorize the issue as soon as possible. If an issue is hard to fix, an edge ca
 reply and put the issue in backlog. You may also encourage the user to contribute a PR themselves if we are unlikely to
 find time to resolve the issue ourselves anytime soon.
 
+Additionally, triaging does not have to happen in one sitting. If you've invested a reasonable amount of time into
+triaging an issue, but have not yet found the root cause/a solution, you can always post an update in the issue about
+what you've tried so far (and what worked/didn't work), and continue looking into the issue later/on another day. This
+depends on the severity of the issue, of course - if something appears to be a critical issue potentially affecting lots
```

Comment:
```suggestion
depends on the severity of the issue, of course—if something appears to be a critical issue potentially affecting lots
```
L: I know it might look weird, but this would be a place for an em-dash 😄

Reply:
```suggestion
depends on the severity of the issue, of course — if something appears to be a critical issue potentially affecting lots
```

let's do this?